File: templates/Java/Java7/Procon.java
import java.io.*;

public class <+FILEBASE+> {
	public static void main(String[] args) throws Exception {
		try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
			String line = null;
			while ((line = br.readLine()) != null) {
				String[] tokens = line.split(" ");
				<+CURSOR+>
			}
		}
	}
}

File: templates/VisualBasic/Procon.vb
Imports System
Imports System.IO

Public Class MainClass
    Shared Sub Main()
        Console.SetOut(New StreamWriter(Console.OpenStandardOutput()) With { .AutoFlush = false })

        Dim inputtedLine As String = Console.ReadLine()
        While (Not inputtedLine Is Nothing)
            Dim tokens As Integer() = inputtedLine.Split(" ").Select(Function(token) Integer.Parse(token)).ToArray()
            Console.WriteLine("{0} {1}", tokens(0), tokens(1))
            <+CURSOR+>
            inputtedLine = Console.ReadLine()
        End While

        Console.Out.Flush()
    End Sub
End Class

